Lib/functools.py

```diff
@@ -888,6 +888,15 @@ def cache_clear():
     pass
 
 
+################################################################################
+### cache -- simplified access to the infinity cache
+################################################################################
+
+def cache(user_function, /):
```

Comment:
Maybe the `typed` argument could be exposed so the user can set it to true? I understand that it will be set to `False` most of the time but if you rename `lru_cache(maxsize=None)` because an unbounded cache is not really an LRU cache it seems weird to have to use `lru_cache(maxsize=None, typed=True)` if you want an unbounded typed cache for the same raisons.

Reply:
Since the goal is simplification, I'm electing to not expose the *typed* option.  We already have a way to do it.  Also, that option is *very* rarely used.